Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/07-AKS-cluster/aks-cluster.tf

```diff
@@ -98,3 +98,13 @@ resource "azurerm_role_assignment" "aks-to-acr" {
   role_definition_name = "AcrPull"
   principal_id         = module.aks.kubelet_id
 }
+
+# Azure Security Benchmark Policy. This is set a subscription level and should be customized per customer environment
+
+data "azurerm_subscription" "current" {}
+resource "azurerm_subscription_policy_assignment" "azsecurity-benchmark" {
```

Comment:
I wouldn't include this policy here, it doesn't seem related to this scenario at all.  Luckily it's audit only, but still might be a problem for people deploying since it's so hidden (not tied to our resource groups, etc).  I'd just remove this one.  The feedback list had AKS-specific policy anyhow, so you can replace this with something more fitting for the scenario in a future PR.

Reply:
I removed the benchmark, matched baseline with new policy.tf and added win specific policies.